relay-metrics/src/aggregator.rs

```diff
@@ -67,12 +67,16 @@ struct BucketKey {
 }
 
 impl BucketKey {
-    // Create a 64-bit hash of the bucket key using FnvHasher.
-    // This is used for partition key computation and statsd logging.
+    /// Creates a 64-bit hash of the bucket key using FnvHasher.
```

Comment:
```suggestion
    /// Creates a 64-bit hash of the bucket key using FnvHasher.
    ///
```

First line in docs should stand alone